src/components/dialogs/network-modifications/by-formula/formula/formula-form.tsx

```diff
@@ -44,7 +44,9 @@ const FormulaForm: FunctionComponent<FormulaProps> = ({ name, index }) => {
 
     const intl = useIntl();
 
-    const equipmentFields = EQUIPMENTS_FIELDS?.[equipmentTypeWatch] ?? [];
+    const equipmentFields: { id: string; label: string }[] =
+        // @ts-expect-error TODO: conflicts types
+        EQUIPMENTS_FIELDS?.[equipmentTypeWatch] ?? [];
```

Comment:
Maybe you could declare the following types :

> export type EquipmentField = {
    id: string;
    label: string;
};
type EquipmentFields = {
    [key: string]: EquipmentField[];
};

Assign `EquipmentFields` to `EQUIPMENTS_FIELDS` in `formula-utils.tsx` and use `EquipmentField []` where needed, I reckon it would fix those `@ts-expect-error`

Reply:
I don't understand...